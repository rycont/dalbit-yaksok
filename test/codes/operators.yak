"만세 " * 3 보여주기
10 * "벌레" 보여주기
10 >= 5 보여주기
6 <= 3 보여주기