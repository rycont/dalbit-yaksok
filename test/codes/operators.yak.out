만세 만세 만세 
벌레벌레벌레벌레벌레벌레벌레벌레벌레벌레
참
거짓
